src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java

```diff
@@ -82,6 +85,10 @@ public boolean getAddressAmended() {
         return addressAmended;
     }
 
+    public boolean getPhoneAmended() {
+        return phoneAmended;
+    }
```

Comment:
Convention is to use `is` as the prefix for a boolean getter

Reply:
the template engine looks for `get`  when I did `isPhoneAmended` it failed. So defaulted to `get`

Reply:
Okey dokey!

Reply:
thanks @andrewwa-kainos